Jellyfin.Server/Program.cs

```diff
@@ -657,5 +674,8 @@ private static string NormalizeCommandLineArgument(string arg)
 
             return "\"" + arg + "\"";
         }
+
+        [DllImport("libc")]
+        private static extern int chmod(string pathname, int mode);
```

Comment:
This can be moved to L352 to bring it withing the `Unix` block

Reply:
Moved to L355 since it's only being used there.